hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/metrics2/util/TestSampleQuantiles.java

```diff
@@ -92,22 +94,21 @@ public void testClear() throws IOException {
   @Test
   public void testQuantileError() throws IOException {
     final int count = 100000;
-    Random r = new Random(0xDEADDEAD);
     int[] values = new int[count];
     for (int i = 0; i < count; i++) {
       values[i] = i + 1;
     }
 
-    // Do 10 shuffle/insert/check cycles
-    for (int i = 0; i < 10; i++) {
+    // Repeat shuffle/insert/check cycles 10 times
+    for (int i = 0; i < NUM_REPEATS; i++) {
 
       // Shuffle  
-      Collections.shuffle(Arrays.asList(values), r);
+      Collections.shuffle(Arrays.asList(values), rnd);
```

Comment:
Check the checkstyle and empty line warnings.

Reply:
fixed the empty space after comment on L105